modules/group/main.tf

```diff
@@ -23,3 +23,26 @@ resource "aws_iam_group_policy_attachment" "policy_attachments" {
     aws_iam_group.group,
   ]
 }
+
+resource aws_iam_group_policy mfa {
```

Comment:
@jai Why are there no quotes around the resource type and name?

Reply:
fixed